src/main/java/com/xiaomi/infra/pegasus/client/ClientOptions.java

```diff
@@ -75,6 +79,8 @@
   private final String authProtocol;
   private final Credential credential;
   private final long sessionResetTimeWindowSecs;
+  private final int maxFetchCount;
+  private final int maxFetchSize;
```

Comment:
Same, naming `*Bytes` instead, `Count` and `Size` are confused to distinguish.

Reply:
ok